queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
+(call
+  target: (_) @functionCallee) @functionCall @functionCallee.domain
+
+;;(keywords
+;;  (pair
+;;    key: (_) @collectionKey @collectionItem.trailing.end.startOf
+;;    value: (_) @value) @collectionItem @collectionItem.trailing.start.endOf
+;;    (",")? @collectionItem.trailing.start.startOf
+;;    (#allow-multiple! @collectionItem))
+
+;; How does surrounding whitespace work for this
+(map_content
+  (binary_operator
+      left: (_) @collectionKey @collectionKey.trailing.start.endOf
+      right: (_) @value @collectionKey.trailing.end.startOf) @collectionItem
+      ","? @collectionItem.trailing
```

Comment:
this isn't quite right; you're missing the an optional `.leading` as well. And the whitespace is a bit off.  Have a look at https://github.com/cursorless-dev/cursorless/blob/c5ef8fc559302c721dff8dbf25b3501a5f83fbdd/queries/talon.scm#L168-L179. Tho keep in mind that one might behave strangely when there are comments mixed between args. That's still kind of uncharted territory as all this stuff is fairly new. Worth wiring it up and testing it with comments in between the items to see what happens.

Reply:
Is this looking better?

```
;; %{:a => "lorem", "b" => "ipsum", 3 => "dolor"},
(map_content
  (_)? @_.leading.start.endOf
  .
  (binary_operator
    left: (_) @collectionKey
    right: (_) @value
  ) @collectionItem @_.leading.end.startOf @_.trailing.start.endOf
  .
  (_)? @_.trailing.end.startOf
  (#insertion-delimiter! @collectionItem ", ")
)
```

Reply:
not quite, because you only want the outer leading / trailing to be for `collectionItem`, so eg you want `collectionItem.leading.start.endOf` rather than `_.leading.start.endOf`, because `_` is a wildcard

I'd be tempted to give `collectionItem` its own pattern, even if slightly duplicative, to make things easier to read